lunatrace/bsl/backend/src/workers/handler.ts

```diff
@@ -184,21 +184,35 @@ function wrapProcessRepositoryJob(
   };
 }
 
-async function loadQueueHandlers(): Promise<QueueHandlersType> {
-  const webhooks = await createGithubWebhookInterceptor();
-
+function loadQueueHandlers(): QueueHandlersType {
   return {
     'process-manifest': {
-      handlerFunc: wrapProcessS3EventQueueJob(handleSnapshotManifest),
+      // eslint-disable-next-line @typescript-eslint/require-await
+      handlerFunc: async () => {
+        return wrapProcessS3EventQueueJob(handleSnapshotManifest);
+      },
     },
     'process-repository': {
-      handlerFunc: wrapProcessRepositoryJob(generateSnapshotForRepository),
+      // eslint-disable-next-line @typescript-eslint/require-await
+      handlerFunc: async () => {
+        return wrapProcessRepositoryJob(generateSnapshotForRepository);
+      },
     },
     'process-sbom': {
-      handlerFunc: wrapProcessS3EventQueueJob(handleScanSbom),
+      // eslint-disable-next-line @typescript-eslint/require-await
+      handlerFunc: async () => {
```

Comment:
Im not sure why we had to wrap all of these in async.  Maybe in an effort to get async errors to throw?  I have a feeling we could DRY this up by doing what youre doing here in the `readDataFromQueue` function that is taking this callback.  

Again, tiny nit, doesnt need fixing.